go/apps/api/openapi/spec/paths/v2/keys/rerollKey/V2KeysRerollKeyResponseData.yaml

```
@@ -0,0 +1,36 @@
+
+      **SECURITY CRITICAL:**
+      - This is the only time you'll receive the complete key
+      - Unkey stores only a hashed version (unless the original key was created with `recoverable=true`)
+      - Never log, store, or expose this value in your systems
+      - Transmit directly to the end user via secure channels only
+      - If lost and not recoverable, you must reroll or create a new key
+
+      The key format follows: `[prefix]_[random_bytes]`
+      - Prefix is extracted from the original key or uses API default
+      - Random bytes follow API configuration (default: 16 bytes)
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)